code/datums/supplypacks.dm

```diff
@@ -1026,7 +1026,7 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 
 /datum/supply_packs/misc/mule
 	name = "MULEbot Crate"
-	contains = list(/obj/machinery/bot/mulebot)
+	contains = list(/mob/living/simple_animal/bot/mulebot)

```

Comment:
Well emergency crate worked for me. Didn't try the mulebot one. Will check in a moment.